A <%= @exception.class %> occurred in <%= @controller.controller_name %>#<%= @controller.action_name %>:

  <%= @exception.message %>
  <%= @backtrace.first %>
